### ISSUE TITLE
Ruby-command defaults to rbenv, but Bitrise is moving to/suggest asdf first

### ISSUE THREAD

**Issue by broksonic21** [bitrise-io/go-steputils#83]
See [https://github.com/bitrise-io/go-steputils/blob/v1.0.5/command/rubycommand/rubycommand.go#L69-L77](https://github.com/bitrise-io/go-steputils/blob/v1.0.5/command/rubycommand/rubycommand.go#L69-L77)

which prefers rbenv over asdf

However, https://discuss.bitrise.io/t/xcode-14-3-x-edge-stack-update-2023-06/22945 suggests using asdf over Rbenv

but if you do, the RubyCommand still finds rbenv and thinks it needs to use it. This causes steps, like cocoa pods-install (see [https://github.com/bitrise-steplib/steps-cocoapods-install/blob/75dd45d47a09aecdec2f90738ebb78afc677a8dd/main.go#L368 ](https://github.com/bitrise-steplib/steps-cocoapods-install/blob/75dd45d47a09aecdec2f90738ebb78afc677a8dd/main.go#L368 )) to try and install Ruby again even if it's there from asdf

**Comment by lpusok** [comment]
Hello @broksonic21,
Thanks for reporting, a fix is in progress (https://github.com/bitrise-io/go-steputils/pull/81).
Can you confirm that the problem you are seeing is an extra unnecessary installation time for Ruby? Can you share a build URL or part of the build log where this happens, so we can test the fix?

**Comment by broksonic21** [comment]
Yes, we installed a ruby version via asdf, but then the CocoaPods-install step tried to install it again (via rbenv) vs realizing the ruby version was installed and being used already.

I do suggest a slightly different check from your change - is there a reason you aren't checking for ruby version matching (for any reason) before going into the asdf/rbenv checking flow? 

I am concerned on your change, as it will have the _opposite intent_ if you do use rbenv -> as asdf will be checked first, it will try and install even if ruby is installed via rbenv first.

FYI: this wasn't just a timing issue - we saw build failures since rbenv failed to get the proper version due to a network issue.
